src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: "What's new in Astro - December 2023"
+description: "December: last month of the year, a time to reflect on all the amazing things we've shipped, including Astro 4.0!"
+publishDate: "December 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-december-2023/og.webp"
+lang: "en"
+---
+
+## News
+
+- We launched a [$100,000 Astro Ecosystem Fund](https://astro.build/blog/astro-ecosystem-fund/) and have already awarded $10,000 each to three recipients!
+- Sentry is now Astro’s [Official Monitoring Partner](https://astro.build/blog/sentry-official-monitoring-partner/).
+- Astro came top in user satisfaction in [Netlify’s State of Web Development](<https://www.netlify.com/resources/ebooks/the-state-of-web-development-2023/>) report — thanks everyone!
+- [Astro Studio is coming soon…](https://twitter.com/astrodotbuild/status/1733199595231199584)
+
+## Releases
+
+Just in time for the end of the year, we released Astro 4.0!
+
+- [Astro 4.0](/blog/astro-4/)
+- [Starlight 0.15](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.15.0)
+
+## Docs
+
+The [Astro Documentation](https://docs.astro.build) is now running on Starlight!
+
+We have also officially launched [Astro Docs Docs (AD²)](https://contribute.docs.astro.build) to help you contribute to docs! You'll find all kinds of helpful information whether you're making your first open-source PR or are already a seasoned member of Team Docs.
+
+Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a codespace, and contribute to the docs. This month’s topics included:
+
+- Building a new header component for our integrations pages
+- Establishing a community Glossary project
+- Discussing best practices for writing helpful Changelog messages
+- Adding a mini-showcase with links to related themes on our CMS guide pages
+- Updating our integration docs to include MDX components and Expressive Code
+
+## Community
+
+### Community Awards November
+
+- 🌟 $250 awarded to Atharva for significant contributions to Astro Docs and active community involvement.
+- 🥳 $250 awarded to McFlyPartages for translating a significant portion of the Astro Docs into French.
+- 🎉 $250 awarded to Hippo for exceptional work releasing, maintaining, and providing support for Expressive Code.
+- 🍾 $250 awarded to Eva Decker for actively supporting the Astro Community on Discord.
+- 🎊 $250 awarded to Jumper for being a super positive, welcoming and supportive presence in the Astro Discord.
+- 🎁 $250 awarded to Happydev for significant work on the Astro Core code base.
```

Comment:
```suggestion
- 🌟 $250 awarded to [Atharva](https://astro.badg.es/contributor/at-the-vr/) for significant contributions to Astro Docs and active community involvement.
- 🥳 $250 awarded to [McFlyPartages](https://astro.badg.es/contributor/McFlyPartages/) for translating a significant portion of the Astro Docs into French.
- 🎉 $250 awarded to [Hippo](https://astro.badg.es/contributor/hippotastic/) for exceptional work releasing, maintaining, and providing support for Expressive Code.
- 🍾 $250 awarded to [Eva Decker](https://astro.badg.es/contributor/evadecker/) for actively supporting the Astro Community on Discord.
- 🎊 $250 awarded to [Jumper](https://astro.badg.es/contributor/jdtjenkins/) for being a super positive, welcoming and supportive presence in the Astro Discord.
- 🎁 $250 awarded to [Happydev](https://astro.badg.es/contributor/MoustaphaDev/) for significant work on the Astro Core code base.
```
So, I both like hyping up Astro badges AND I like the visual interestingness of having these be links. That's my thinking here!